observability/runtime/config.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2025 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+import (
+	"fmt"
+	"time"
+
+	configmap "knative.dev/pkg/configmap/parser"
+)
+
+const (
+	ProfilingEnabled  = "enabled"
+	ProfilingDisabled = "disabled"
+)
+
+type Config struct {
+	Profiling      string
+	ExportInterval time.Duration
+}
+
+func (c *Config) Validate() error {
+	switch c.Profiling {
+	case ProfilingEnabled, ProfilingDisabled:
+	default:
+		return fmt.Errorf("unsupported profile setting %q", c.Profiling)
+	}
+
+	if c.ExportInterval < 0 {
```

Comment:
Should the export interval be allowed to be 0?
```suggestion
	if c.ExportInterval <= 0 {
```

Reply:
yup - it will use the OTel default